thinc/backends/_custom_kernels.py

```diff
@@ -541,7 +555,7 @@ def backprop_reduce_mean(d_means, lengths, *, threads_per_block=128, num_blocks=
     O = d_means.shape[1]
     _check_lengths(lengths, T)
 
-    out = cupy.zeros((T, O), dtype=d_means.dtype)
+    out = _alloc((T, O), dtype=d_means.dtype, zeros=True)
```

Comment:
`backprop_reduce_mean` should backprop to every sequence item, so I think this can be `zeros=False`.